editmenu/UndoFunction.java

```diff
@@ -0,0 +1,21 @@
+package editmenu;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JTextPane;
+
+public class UndoFunction implements ActionListener{
+	private JTextPane textPane;
+	
+	public UndoFunction(JTextPane textPane) {
+		this.textPane = textPane;
+	}
+
```

Comment:
If ActionPerformed is not used, you can remove implements ActionListener from the class as a whole